scene/gui/rich_text_label.cpp

```diff
@@ -1641,6 +1650,40 @@ void RichTextLabel::_notification(int p_what) {
 				RenderingServer::get_singleton()->canvas_item_add_clip_ignore(ci, false);
 			}
 
+			bool ready = _validate_line_caches();
+			if (ready) {
+				ready = _update_scroll();
+			}
+
+			if (!ready) {
```

Comment:
Maybe add a short comment like "// Draw a progress bar if we're still processing."
Some users rightfully complain that we're a bit too terse on comments, some general comments giving overview on what a block does can be useful.